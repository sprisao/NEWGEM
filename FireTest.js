import {StyleSheet, Text, View} from "react-native";
import React from "react";

const FireTest = () => {
  return (
    <View>
      <Text>Practice TDD</Text>
    </View>
  );
};

export default FireTest;

const styles = StyleSheet.create({});
