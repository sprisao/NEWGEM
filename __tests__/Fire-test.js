import React from "react";
import renderer from "react-test-renderer";
import FireTest from "../FireTest";

describe("파이어테스트", () => {
  test("화면이 정상적으로 들어오는가", () => {
    const screen = renderer.create(<FireTest />).toJSON();
    expect(screen).toMatchSnapshot();
  });
  test("타이틀이 정상적으로 보여지는가", () => {
    const screen = renderer.create(<FireTest />).toJSON();
    const title = screen.getByText("Practice TDD");
    expect(title).toBeDefined();
  });
});
//   test("화면이 정상적으로 들어오는가", () => {
//     const screen = renderer.create(<FireTest />).toJSON();
//     expect(screen).toMatchSnapshot();
//   }),
//   test("화면이 정상적으로 들어오는가", () => {
//     const screen = renderer.create(<FireTest />).toJSON();
//     expect(screen).toMatchSnapshot();
//   }),
//   test("화면이 정상적으로 들어오는가", () => {
//     const screen = renderer.create(<FireTest />).toJSON();
//     expect(screen).toMatchSnapshot();
//   }),
