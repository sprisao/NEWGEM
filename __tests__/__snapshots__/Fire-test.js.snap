// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`파이어테스트 화면이 정상적으로 들어오는가 1`] = `
<View>
  <Text>
    Practice TDD
  </Text>
</View>
`;
